.github/dependabot.yml

```
@@ -0,0 +1,47 @@
+# https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+
+updates:
+  - package-ecosystem: "npm"
+
+    directory: "/"
+
+    schedule:
+      interval: "daily"
```

```suggestion
      interval: "monthly"
```

This is not a sensitive package, let's not consume too many earth resources